ls1intum/Artemis
# Replace template placeholders during import of Programming Exercise.

### Checklist

### Motivation and Context



### Description


### Steps for Testing


1. Create a programming exercise from a template
2. (Make changes to the template)
3. Import the created programming exercise into a new programming exercise
4. Verify that a commit in the base, solution, and test repository of the new exercise, that replaces all occurrences of the old exercise title with the new name

See the screencast if you are unsure on how to do that:
![Peek 2020-10-21 01-05](https://user-images.githubusercontent.com/5084100/96653631-d0147f80-1339-11eb-89a3-cc3c6964bf04.gif)


### Test Coverage

* ExerciseService.java: 85%
* programming-exercise.component.ts 95%

### Screenshots
